INSTALL

```diff
@@ -33,6 +33,11 @@ some of the steps manually.  The more detailed description in the other
 sections of this guide will help you do that, so please refer to those
 sections if you need to.
 
+  0. Before you start: Remacs uses cargo for it's build process. if
+     you build as an unpriviledged user and then install as an
+     administrator cargo will rebuild remacs. Make sure you have the
+     right nightly version installed for both users.
+
```

Comment:
Maybe mention `rust-toolchain` as the source of that version?

Reply:
hm, not sure how elaborate to be here,
should i just mention rustc or refer it?

Reply:
How about

"Make sure you have the right nightly version as found in the rust-toolchain file." 

Or something like that.